packages/jest-emotion/test/react-enzyme.test.js

```diff
@@ -1,130 +1,216 @@
-import 'test-utils/legacy-env'
+import 'test-utils/legacy-env' // adds serializer
 /** @jsx jsx */
+import jestInCase from 'jest-in-case'
 import * as enzyme from 'enzyme'
-import { jsx } from '@emotion/core'
-import { createSerializer as createEnzymeSerializer } from 'enzyme-to-json'
-import { createSerializer } from 'jest-emotion'
-import { toMatchSnapshot } from 'jest-snapshot'
+import { jsx, ThemeProvider } from '@emotion/core'
+import styled from '@emotion/styled'
 import React from 'react'
+import toJson from 'enzyme-to-json'
 
-const createEnzymeSnapshotMatcher = serializerOptions => {
-  const serializer = createEnzymeSerializer(serializerOptions)
-  const identityPrinter = v => v
+afterEach(() => {
+  document
+    .querySelectorAll('style[data-emotion]')
+    .forEach(element => element.parentNode.removeChild(element))
+})
 
-  return function(val) {
-    return toMatchSnapshot.call(this, serializer.print(val, identityPrinter))
-  }
+const tickle = (wrapper: *) => {
```

Comment:
Is our consumer supposed to do that on their own?

Reply:
There's two basic options here if we want to publicize exposing the styles in the snapshot for nested components - either we just document this approach, or we expose a helper and document how to use it.

A more ambitious option would be to wrap the underlying `jest-enzyme` implementation. If we take that approach, then users would use `jest-emotion` in place of `jest-enzyme`, instead of using them both together.

Reply:
or maybe a sub-package, like `jest-emotion/enzyme`

Reply:
sub-package sounds good, to create one you can define it like this:
https://github.com/emotion-js/emotion/blob/2fa7a213222fc2bb99ca0a01078148f1a1c6458d/packages/styled/package.json#L55-L58